src/clients/mod.rs

```diff
@@ -84,16 +59,28 @@ mod test {
             ..Default::default()
         };
 
-        let (auth, value) = bearer_auth(&tok);
-        assert_eq!(auth, "authorization");
-        assert_eq!(value, "Bearer access_token");
+        let headers = tok.auth_headers();
```

Comment:
Same here, this test should be moved into `rspotify-model/src/auth.rs`, the same file with `struct Token`

Reply:
Yup agreed